opengrok-indexer/src/main/java/org/opengrok/indexer/index/PendingFileCompleter.java

```diff
@@ -171,6 +185,15 @@ public boolean add(PendingFileRenaming e) {
      * @throws java.io.IOException if an I/O error occurs
      */
     public int complete() throws IOException {
+        completing = true;
+        try {
+            return complete2();
+        } finally {
+            completing = false;
+        }
+    }
+
+    private int complete2() throws IOException {
```

Comment:
use some more descriptive name ?

Reply:
I used to name these like `completeInner()` or `completeWork()` but I now like to keep the name even closer to the public method, and the single `2` character is the smallest difference, yet still communicates sequence. Thoughts?

Reply:
perhaps call the outer method `completeWrapper()` ?

Reply:
actually, `complete()` and `completeInner()` would be fine to me as well.

Reply:
Also, I saw somewhere naming scheme such as `complete()` and `doComplete()`.

Reply:
done as `completeInner()`